electron/execution.ts

```diff
@@ -191,11 +199,11 @@ async function onTest(data, browserWindow) {
     }
   };
 
-  const sendTestEvent = event => {
+  const sendTestEvent = (event: TestEvent) => {
     browserWindow.webContents.send('test-event', event);
   };
 
-  const emitResult = chunk => {
+  const emitResult = (chunk: any) => {
```

Comment:
```suggestion
  const emitResult = (chunk: string) => {
```

I think `chunk` is a `string`, right?

Reply:
It's either Buffer or string, but we know that Synthetics will emit string so I guess we can say it's string.

Reply:
I'm fine with annotating it as `Buffer | string`, either option to me is better than explicitly naming it with `any` if we can avoid it.